examples/inspection/plot_permutation_importance_multicollinear.py

```diff
@@ -8,26 +8,29 @@
 The :class:`~sklearn.ensemble.RandomForestClassifier` can easily get about 97%
 accuracy on a test dataset. Because this dataset contains multicollinear
 features, the permutation importance will show that none of the features are
-important. One approach to handling multicollinearity is by performing
-hierarchical clustering on the features' Spearman rank-order correlations,
-picking a threshold, and keeping a single feature from each cluster.
+important. Hierarchical clustering is used to visualize the correlations
+between features and :class:`~sklearn.feature_selection.CorrelationThreshold`
+is used to ensure all pairwises correlations are below a certain threshold.
```

Comment:
`... to filter the features in a way that ...` may be better. "to ensure" may be a bit vague here.